config/release-package-matrix.js

```diff
@@ -13,92 +13,92 @@ exports.RELEASE_PACKAGE_MATRIX = [
     executableOsId: 'darwin-x64',
     compileBuildVariant: 'darwin',
     packages: [
-      { name: 'darwin-x64', description: 'MacOS 64-bit (10.14+)', packageOn: 'darwin', smokeTestKind: 'macos', serverLikeTargetList: ['macos'] }
+      { name: 'darwin-x64', description: 'MacOS x64 (10.14+)', packageType: 'zip', packageOn: 'darwin', smokeTestKind: 'macos', serverLikeTargetList: ['macos'] }
     ]
   },
   {
     executableOsId: 'darwin-arm64',
     compileBuildVariant: 'darwin_arm64',
     packages: [
-      { name: 'darwin-arm64', description: 'MacOS M1 (11.0+)', packageOn: 'darwin', smokeTestKind: 'macos', serverLikeTargetList: ['macos'] }
+      { name: 'darwin-arm64', description: 'MacOS M1 (11.0+)', packageType: 'zip', packageOn: 'darwin', smokeTestKind: 'macos', serverLikeTargetList: ['macos'] }
     ]
   },
   {
     executableOsId: 'linux-x64',
     compileBuildVariant: 'linux_x64_build',
     packages: [
-      { name: 'linux-x64', description: 'Linux Tarball 64-bit', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu20.04-tgz'], serverLikeTargetList: [...allLinux] },
-      { name: 'deb-x64', description: 'Debian / Ubuntu 64-bit', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu18.04-deb', 'ubuntu20.04-deb', 'ubuntu22.04-deb', 'ubuntu22.04-nohome-deb', 'debian9-deb', 'debian10-deb', 'debian11-deb', 'debian12-deb'], serverLikeTargetList: [...ubuntu1804AndAboveAndDebBased] },
-      { name: 'rpm-x64', description: 'RHEL / CentOS / Fedora / Suse 64-bit', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['centos7-rpm', 'amazonlinux2-rpm', 'amazonlinux2023-rpm', 'rocky8-rpm', 'rocky9-rpm', 'fedora34-rpm', 'suse12-rpm', 'suse15-rpm', 'amazonlinux1-rpm'], serverLikeTargetList: [...rhel70AndAboveAndRpmBased] }
+      { name: 'linux-x64', description: 'Linux x64', packageType: 'tgz', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu20.04-tgz'], serverLikeTargetList: [...allLinux] },
+      { name: 'deb-x64', description: 'Debian (10+) / Ubuntu (18.04+) x64', packageType: 'deb', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu18.04-deb', 'ubuntu20.04-deb', 'ubuntu22.04-deb', 'ubuntu22.04-nohome-deb', 'debian9-deb', 'debian10-deb', 'debian11-deb', 'debian12-deb'], serverLikeTargetList: [...ubuntu1804AndAboveAndDebBased] },
```

Comment:
Just a minor suggestion, maybe we could assign common descriptions to a variable at the top of the file, e.g. `const debianUbuntuDescription = 'Debian (10+) / Ubuntu (18.04+) x64';` and then use that here (and ditto for others of course), that would make it harder to accidentally un-group packages through typos in future edits?